README.rst

```diff
@@ -350,8 +350,10 @@ Modeler Plugins
 
 .. Note::
    As of version 2.3.0 the zenoss.cmd.linux.rpm and zenoss.cmd.linux.alt\_kernel\_name
-   modeler plugins are disabled by default. It is recommended you disable the modeler plugin
-   zenoss.cmd.linux.alt\_kernel\_name if you have a customized etc/issue file.
+   modeler plugins are disabled by default on new installs. If upgrading from
+   a version previous to 2.3.0 they will still be enabled by default. It is
+   recommended you disable the modeler plugin zenoss.cmd.linux.alt\_kernel\_name
+   if you have a customized etc/issue file.
```

Comment:
Add an extra / to etc/issue and explain why a modified /etc/issue is a problem for the modeler.